test/python/test_ripple.py

```diff
@@ -186,7 +186,7 @@ def test_ripple_swap_refuse_double_sign(backend, navigator, test_name):
     performer.perform_valid_swap(backend, navigator, test_name)
     performer.perform_ripple_tx(backend)
 
-    with pytest.raises((ChunkedEncodingError, ConnectionError, ProtocolError, IncompleteRead)):
+    with pytest.raises((ChunkedEncodingError, ConnectionError, ProtocolError, IncompleteRead, OSError)):
         performer.perform_ripple_tx(backend)
```

Comment:
For `OSError`, assert that we receive the correct BOLOS APDU

Reply:
That was possible with previous implem which was expecting an APDUException
This not the case anymore as I removed some magic from the ledgerwallet.
This is now raising an OSError except if we do a handle_usb_reset first.